grab_ss.py

```diff
@@ -0,0 +1,77 @@
+from Xlib import display
+import numpy as np
+from PIL import Image, ImageGrab
+import subprocess
+import time
+
+def find_window_id(window_name):
+    root = display.Display().screen().root
+    window_id = None
+
```

Comment:
why I this screenshot grabbing different then the other files we have?

Reply:
This actually is my fault. Screen grabbing wasn't working well on this laptop, So, I was trying different methods. This shouldn't have been committed.